discord.html

```diff
@@ -1,200 +1,186 @@
+              <ul class="list list-reset">
+                <li class="category-name">
+                  <strong>Live: </strong>Which lets you connect with the
+                  members.You can seperatly get connected on call with members
+                  of specific channel.
+                </li>
 
-                  <li class="category-name">
-                    <strong>Admin: </strong> All Channels related to admin.
-                  </li>
+                <li class="category-name">
+                  <strong>Admin: </strong> All Channels related to admin.
+                </li>
 
-                  <li class="">
-                    <strong>Main: </strong> It has channels for general
-                    disussion,progress-updates ,newcomers & Support.
-                    <ol class="channel-info">
-                      <li class="channel-name">
-                        general:It carries general discussions.
-                      </li>
-                      <li class="channel-name">
-                        newcomers:All info related to new commers are floated in
-                        it.
-                      </li>
-                      <li class="channel-name">
-                        progress-updates:All info related to current ongoing
-                        task,discussions are posted here.
-                      </li>
-                      <li class="channel-name">
-                        support:In case you are stuck on need some help everyone
-                        is there to help you keep running.
-                      </li>
-                    </ol>
-                  </li>
+                <li class="">
+                  <strong>Main: </strong> It has channels for general
+                  disussion,progress-updates ,newcomers & Support.
+                  <ol class="channel-info">
+                    <li class="channel-name">
+                      general:It carries general discussions.
+                    </li>
+                    <li class="channel-name">
+                      newcomers:All info related to new commers are floated in
+                      it.
+                    </li>
+                    <li class="channel-name">
+                      progress-updates:All info related to current ongoing
+                      task,discussions are posted here.
+                    </li>
+                    <li class="channel-name">
+                      support:In case you are stuck on need some help everyone
+                      is there to help you keep running.
+                    </li>
+                  </ol>
+                </li>
 
-                  <li class="category-name">
-                    <strong>Text:</strong> Contains multiple other channels in
-                    it.
-                  </li>
+                <li class="category-name">
+                  <strong>Text:</strong> Contains multiple other channels in
```

Comment:
Can we change `Text` to `Text Channels` like we have it in our discord server ?

Reply:
fixed

Reply:
I think you changed it at different place. 😅

Reply:
done